valuesets/ValueSet-England-PeriodType.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+	<id value="England-PeriodType"/>
+	<url value="https://fhir.nhs.uk/England/ValueSet/England-PeriodType"/>
+	<version value="1.0.0"/>
+	<name value="EnglandPeriodType"/>
+	<title value="England Period Type"/>
+	<status value="draft"/>
+	<date value="2023-10-13"/>
+	<publisher value="NHS England" />
+	<contact>
+		<name value="NHS England" />
+		<telecom>
+			<system value="email" />
+			<value value="interoperabilityteam@nhs.net" />
+			<use value="work" />
+			<rank value="1" />
+		</telecom>
+	</contact>
+	<description value="A ValueSet to identify the type of period data being recorded."/>
```

Comment:
Doesnt read right imo. Maybe "A ValueSet to identify the type of data for the date/time period described within the extension."

Reply:
Agree, changed